library/common/extensions/filters/http/test_event_tracker/config.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <string>
+
+#include "source/extensions/filters/http/common/factory_base.h"
+
+#include "library/common/extensions/filters/http/test_event_tracker/filter.h"
+#include "library/common/extensions/filters/http/test_event_tracker/filter.pb.h"
+#include "library/common/extensions/filters/http/test_event_tracker/filter.pb.validate.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace TestEventTracker {
+
+/**
+ * Config registration for the EventTracker filter. @see NamedHttpFilterConfigFactory.
+ */
+class TestEventTrackerFilterFactory
+    : public Common::FactoryBase<
+          envoymobile::extensions::filters::http::test_event_tracker::TestEventTracker> {
+public:
+  TestEventTrackerFilterFactory() : FactoryBase("test_event_tracker") {}
+
+private:
+  ::Envoy::Http::FilterFactoryCb createFilterFactoryFromProtoTyped(
```

Comment:
what's `Cb`?

Reply:
This is just an alias on a function that comes from Envoy repo [here](https://github.com/envoyproxy/envoy/blob/main/envoy/http/filter.h#L1049-L1056). The function is used to create filter - I think it's something like a factory but implemented with the use of a function that's returned here and should be called every time a new instance of a filter is needed. 

Frankly speaking, the only reason why I know about this type is because I used `TestAccessor` as an example for my changes and it uses `Cb` in [here](https://github.com/envoyproxy/envoy-mobile/blob/main/library/common/extensions/filters/http/test_accessor/config.h#L26)

Reply:
ah Cb probably means callbacks

Reply:
It means callbacks :)

Reply:
haha, I thought that we had `Ca` somewhere `Cb` was just next available name - lol